web/modules/custom/dpl_example_content/content/eventseries/2ce9d9b0-7f10-4a17-9a5e-383d36c1ed93.yml

```diff
@@ -100,34 +101,37 @@ default:
         href: '/user/login'
   event_instances:
     -
-      entity: 21850bf7-ffc9-483c-bcd5-489d420552f9
+      entity: 744dba3f-0fbc-4f58-81ca-7d99a862f4f7
+  field_branch:
+    -
+      entity: 852d781f-82c9-40ac-bc8d-d0cefe07706d
```

Comment:
:+1: for remembering to create/update example content with new features as we add them.